arch/risc-v/src/mpfs/Kconfig

```diff
@@ -250,6 +250,33 @@ config MPFS_EMMCSD
 	---help---
 		Selects the MPFS eMMCSD driver.
 
+choice
+	depends on MPFS_EMMCSD && MMCSD_MMCSUPPORT
+	prompt "Configure eMMC clock speed"
+	---help---
+		Select the desired eMMC clock frequency.  This doesn't issue any CMD6 commands to
+		switch the mode at the moment.  This only changes the eMMC clock.  Not all hardware
+		support 200 MHz but the Polarfire Icicle kit does.
+
+config MPFS_EMMC_CLK_200MHZ
+	bool "EMMC CLK 200MHz"
+	default y
+	---help---
+		Set this to use 200MHz eMMC clock
+
+config MPFS_EMMC_CLK_50MHZ
+	bool "EMMC CLK 50MHz"
+	default n
+	---help---
+		Set this to use 50MHz eMMC clock
+
+config MPFS_EMMC_CLK_26MHZ
+	bool "EMMC CLK 26MHz"
+	default n
+	---help---
+		Set this to use 26MHz eMMC clock
+endchoice
```

Comment:
Maybe this can be just a define from `board.h`? I mean that IMO clocking selection is very similar to GPIO selection and can be handled in a similar way. What do you think?

Reply:
In this case it'd simplify things, I agree.

Reply:
Changed